Script/AtomicEditor/ui/AnimationToolbar.ts

```diff
@@ -170,12 +170,14 @@ class AnimationToolbar extends Atomic.UIWidget {
 
         this.animatedModel = <Atomic.AnimatedModel>modelNode.getComponent("AnimatedModel");
         this.animationController = <Atomic.AnimationController>modelNode.getComponent("AnimationController");
-        var model = this.animatedModel.model;
-        this.animatedModel.setBoneCreationOverride(true);
-        this.animatedModel.setModel(model, true);
+        if ( this.animatedModel != null && this.animationController != null ) {
+            var model = this.animatedModel.model;
```

Comment:
Good catch, thanks!  I am sure we'll have at least one patch for Build 3 this can make it into, I wanted to put binaries up that had been tested, otherwise would have built this out on CI for initial release.  The caravan must roll on!